crates/bevy_transform/src/systems.rs

```diff
@@ -13,6 +8,7 @@ pub fn transform_propagate_system(
     mut root_query: Query<
         (
             Option<&Children>,
+            Option<Changed<Children>>,
```

Comment:
Instead of `Option<Changed<Children>>`, `ChangeTrackers<Children>` should be used.

Reply:
Why? I thought we'd want to support roots without children - this would be quite a serious regression.

I could change it to `Option<ChangeTrackers<Children>>` of course if you prefer, although I would have thought that getting the `changed` only would be easier.

Reply:
Hm, then i suppose using `Option<ChangeTrackers<Children>>` would be better. I'd rather not mix filters in with stuff meant to be queried for, given the effect mixing other filters like `With` can have.

Reply:
I'm not sure I understand the concern

`Changed` is merely a query which returns true if the component value has changed since the last time the system was run.

Reply:
Maybe, but it's introduced and used as a filter most of the time. I feel like it's bad practice to mix filters with regular queries, given filters like `With` do very different things.

Reply:
I would also like to forbid the use of filters as ordinary query parameters down the line if we can; and this will be one more case that we'd have to revert.

That said, I don't feel strongly, and won't block on it.